### ISSUE TITLE
naming structures not working

### ISSUE THREAD

**Issue by matix48** [DirtyRacer1337/Jellyfin.Plugin.PhoenixAdult#198]
Hello,

I'm doing some test with few videos I have and even by following the naming structures recommended, most of the videos aren't detected. 
![Capture d’écran de 2023-03-09 00-41-33](https://user-images.githubusercontent.com/127339024/223877883-2132e8ff-101a-4183-b305-8eb3481c0bcf.png)
![Capture d’écran de 2023-03-09 00-42-38](https://user-images.githubusercontent.com/127339024/223877991-6f6e907f-a463-4f6b-ac37-769aa03e4b88.png)

Only one video is detected (even if the actors profiles are empty when you click on them)
![Capture d’écran de 2023-03-09 00-43-05](https://user-images.githubusercontent.com/127339024/223878027-d6210943-12e7-490a-a5c6-90a7525f396e.png)

In my folder, the files are like this: (even when I remove the [HD] (year) tags, it doesn't change anything)
![Capture d’écran de 2023-03-09 00-44-04](https://user-images.githubusercontent.com/127339024/223878244-d0c35eb6-81a9-4f0d-87bc-827f294a1696.png)

Did I miss something ?

**Comment by DirtyRacer1337** [comment]
Try dev build

**Comment by matix48** [comment]
I installed the dev build. One video got detected and all the infos appeared perfectly 

![Capture d’écran de 2023-03-09 15-28-08](https://user-images.githubusercontent.com/127339024/224055827-1e575db1-4cb8-4341-a036-3d04d34418b0.png)

However, I noticed something. For the scenes that weren't detected (after trying all the naming structures), if I use the movie name alone, the movie cover appears (not the infos). 

![Capture d’écran de 2023-03-09 15-27-55](https://user-images.githubusercontent.com/127339024/224082595-af23d3a7-c7b3-4750-9f7c-393ec5e5fa71.png)
![Capture d’écran de 2023-03-09 15-28-18](https://user-images.githubusercontent.com/127339024/224082752-cd8c7304-e02f-490b-8e7e-a23a7f26e9ee.png)
Once I add "SiteName - Scene Name" in the title, nothing shows up. 
Another thing I noticed is that for those videos, the PhoenixAdult tag isn't visible, there is imdb instead.
